### DISCUSSION TITLE
目的について

### DISCUSSION THREAD

**Discussion by Honekatsu** [PeaCH-ktq/PeaCH-ktq#7] [General]
サークルの目的について議論する場所です

**Comment by Honekatsu** [comment]
とりあえず思いつくものを並べていきます．サークル，個人両方の立場ごちゃまぜですが．．．
- メンバー個人の開発能力を高めるため
- チーム開発力を身につけるため
- 実際の開発を体験した，即戦力となる人材を育てるため．
- 学生自らがアイデアを出して制作する機会を増やすことで，スタートアップのきっかけを増やす．

**Comment by ushed** [comment]
お互いに高め合うため

**Comment by suzuka-y** [comment]
- メンバーの創作意欲を昇華させ，ソフトウェア開発の能力の向上を図る．
- チーム共同開発を体験し，協調性のあるソフトウェア開発を遂行できるようにする．
- ソフトウェア開発に限らず，同じ分野に関心を持つメンバー間の協調・競争を誘発し，有意義な学生生活を送れるようにする．

**Comment by ushed** [comment]
・創作活動をしている、興味がある学生が集うコミュニティの形成
・本サークルでは創作意欲のある学生が、初めの一歩を踏み出せるような支援
・同じ志をもつ者でのチーム作成
